src/components/Button/Button.scss

```diff
@@ -0,0 +1,37 @@
+.button {
+    justify-self: center;
+    border-radius: 5px;
+    border-width: 0px;
+    background-color: white;
+    font-weight: bold;
+    background-repeat: no-repeat;
+    background-position: center;
+      
+  
+    &--plus{

```

Comment:
this variant knows more than it is needed.
I mean, this is only to decide which icon you'll add here so this class doesn't need to know the size or color, the only goal is to put in here a .svg icon.

My suggestion is to create a separate variant class to handle the small buttons:

```scss
&--small {
  min-width: 40px; // we can control this using padding, so no matter the size of the text the padding-inline (left and right) will be "for small buttons".
  height: 40px;
}
```

The `color/background` for default button could be direct in `.button`:

```scss
.button {
  justify-self: center; // what if you use text-align: center?
  border-radius: 5px;
  border-width: 0px;
  background-color: #F7F8FD;
  font-weight: bold;
  background-repeat: no-repeat;
  background-position: center;
  color: #EC823E;

   // ...
}
```

and then you create a variant only to add the icons image:

```scss
&--minus {
  background-image: url('../../images/icon-minus.svg');
}
&--plus {
  background-image: url('../../images/icon-plus.svg');
}
&--cart {
  background-position: left;
  background-image: url("../../images/icon-cart.svg");
}

```